src/platform/user/profile/vap-svc/util/transactions.js

```diff
@@ -76,6 +76,10 @@ export function isFailedTransaction(transaction) {
   return FAILURE_STATUSES.has(transaction?.data.attributes.transactionStatus);
 }
 
+export function failedTransactionStatus(transaction) {
+  return FAILURE_STATUSES[(transaction?.data.attributes.transactionStatus)];
```

Comment:
My linter adds the `(` and `)` 🤷

Reply:
This is going to be one of only two values and will likely be `COMPLETED_FAILURE` most of the time. So we're going to want to record more specific data.

This is an example of an error we get when a transaction fails:
```json
{
  "data": {
    "id": "",
    "type": "async_transaction_vet360_address_transactions",
    "attributes": {
      "transactionId": "0f7bd0c2-3ccb-4d90-8c45-44a2c9d5c216",
      "transactionStatus": "COMPLETED_FAILURE",
      "type": "AsyncTransaction::Vet360::AddressTransaction",
      "metadata": [
        {
          "code": "ADDRVAL206",
          "key": "addressBio.MultipleAddressError",
          "retryable": null,
          "severity": "ERROR",
          "text": "Multiple matches found for this address"
        }
      ]
    }
  }
}
```

Recording just the `code` might be good enough. I think we could easily ask VA Profile what those codes mean.